src/aws/models/governance-proposals.model.ts

```diff
@@ -0,0 +1,70 @@
+import { embed } from '@aws/dynamodb-data-mapper';
+import { attribute, hashKey, rangeKey, table } from '@aws/dynamodb-data-mapper-annotations';
+import { Network } from '@badger-dao/sdk';
+
+import { GOVERNANCE_PROPOSALS_DATA } from '../../config/constants';
+import { GovernanceProposalsDisputes } from './governance-proposals-disputes.interface';
+import { GovernanceProposalsStatuses } from './governance-proposals-statuses.interface';
+
+@table(GOVERNANCE_PROPOSALS_DATA)
+export class GovernanceProposals {
```

Comment:
am i missing something here. does this account for proposals with more than 1 command? + ScheduleBatch()

Reply:
hmmm, idk, havent unpacked calldata yet, but as i can see, its `bytes`, not `bytes[]`
```
    /**
     * @dev Emitted when a call is scheduled as part of operation `id`.
     */
    event CallScheduled(
        bytes32 indexed id,
        uint256 indexed index,
        address target,
        uint256 value,
        bytes data,
        bytes32 predecessor,
        uint256 readyTime,
        address sender,
        string status
    );
```

Reply:
About ScheduleBatch ye sure, it emits the same event as regular schedule, so should do

Reply:
ah right, if they are part of a batch their id will all be the same, with the index showing the order